doc/en/components/general-getting-started.md

```diff
@@ -355,15 +388,13 @@ We are now ready to use the {ProductName} map component in our markup! Let's go
 
 ```tsx
 // App.txs
-render() {
-    return (
-        <div style={{height: "100%", width: "100%" }}>
-            <IgrGeographicMap
-            width="800px"
-            height="500px"
-            zoomable="true" />
-        </div>
-    );
+function App() {
+  return (
+    <div style={{ height: "100%", width: "100%" }}>
+      <IgrGrid id="grid1" data={localData} autoGenerate="true" />
+      <IgrGeographicMap width="800px" height="500px" zoomable="true" />
+    </div>
+  );
```

Comment:
why are we adding a IG grid without instruction how to define actual data? Getting started needs to be as simple as possible so please keep just GeoMap.

Reply:
All comments should be resolved now